architecture/2-UserStories/5-PayloadScheduler.yaml

```diff
@@ -0,0 +1,9 @@
+id: 5.0
+name: Payload Scheduler
+actor: Ground Station Operator 
+behavior: Schedule data recording at specfic time interval (or orbital location)
+rationale: Recording data for future transmission
+derivedFrom: []
+example: 
+- Ex1 Schedule data recording at 4min -10min (for a given orbital location) for ODU Payload?
+- Ex2 Schedule data recording at specfic time interval (or orbital location) for ODU Payload?
```

Comment:
I'm thinking the second example should be a separate user story all together; i.e., separate scheduling by time interval from scheduling by orbital location

Reply:
This also applies for `architecture/2-UserStories/5.1-ODUPayloadScheduler.yaml` and `architecture/2-UserStories/5.2-CGAPayloadScheduler.yaml`